sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/JsonExpressionsSuite.scala

```diff
@@ -790,4 +790,27 @@ class JsonExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper with
         checkDecimalInfer(_, """struct<d:decimal(7,3)>""")
     }
   }
+
+  test("Length of JSON array") {
+    Seq(
+      ("""""", null),
+      ("""[1,2,3]""", 3),
+      ("""[]""", 0),
+      ("""[[1],[2,3],[]]""", 3),
```

Comment:
This is different from the example I gave you. Do you have any reason to prefer `"""` here? Usually, Apache Spark prefer to use a simple form `"` than `"""`.
- https://github.com/apache/spark/pull/27759#discussion_r400594807
```
Seq(
  ("", null),
  ("[]", 0),
  ("[1,2,3]", 3),
  ("[[1],[2,3],[]]", 3),
```